Sources/Plasma/FeatureLib/pfPython/pyGUIDialog.cpp

```diff
@@ -246,6 +279,26 @@ PyObject* pyGUIDialog::GetControl( uint32_t idx )
     PYTHON_RETURN_ERROR;
 }
 
+PyObject* pyGUIDialog::GetControlMod(uint32_t tagID) const
```

Comment:
This is control index, not tag ID, right?

Reply:
Good point, I'll fix this.